components/switch/script.js

```diff
@@ -0,0 +1,110 @@
+import components from 'coherent-gameface-components';
+import textInside from './templates/text-inside-template.html';
+import textOutside from './templates/text-outside-template.html';
+
+/**
+ * @class Switch
+ * Switch component, that allows you to switch between true and false
+ */
+class Switch extends HTMLElement {
+    constructor() {
+        super();
+
+        this._onClick = this._onClick.bind(this);
+    }
+
+    /**
+     * Called when the element is attached to the DOM
+     */
+    connectedCallback() {
+        this._disabled = this.hasAttribute('disabled');
+        this._checked = this.hasAttribute('checked');
```

Comment:
Usually a checkbox is used for a switch component. Because of this the switch button has a public property "checked". Right now with this naming convention it looks like the _checked property is inaccessible.  We need to provide a way to
check the value of the switch - checked, unchecked.

Reply:
**_** were removed.